catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/SearchController.java

```diff
@@ -498,6 +498,7 @@ private JSONArray getActions(Metacard metacard) {
         List<Action> actions = actionRegistry.list(metacard);
         for (Action action : actions) {
             JSONObject actionJson = new JSONObject();
+            actionJson.put(Search.ACTIONS_ID, action.getId() + action.getTitle());
             actionJson.put(Search.ACTIONS_ID, action.getId());
```

Comment:
Did you forget to remove the other "put" into this map with the same key as the one that you just added?